app/schema.graphql

```diff
@@ -1216,6 +1216,7 @@ type TimeSeriesDataPoint {
 type Trace implements Node {
   """The Globally Unique ID of this object"""
   id: GlobalID!
+  project: Project
```

Comment:
Can't this be Required? I thought? Not blocking

Reply:
yea...but the SQL itself is non-determinant

Reply:
i can change